lib/active_job/queue_adapters/solid_queue_adapter.rb

```diff
@@ -8,10 +8,6 @@ module QueueAdapters
     #
     #   Rails.application.config.active_job.queue_adapter = :solid_queue
     class SolidQueueAdapter
-      def enqueue_after_transaction_commit?
-        true
-      end
```

Comment:
I think we need to leave this in place, unfortunately, because Rails 7.2 relied on this being defined in adapters, so removing it will break for people using Rails < 8.

Reply:
Done!